common/views/components/SearchFilters/SearchFiltersDesktop.tsx

```diff
@@ -238,6 +238,7 @@ const DynamicFilterArray = ({
             changeHandler={changeHandler}
             form={showMoreFiltersModal ? undefined : searchFormId}
             isNewStyle={isNewStyle}
+            isDisabled={!f.options.length}
```

Comment:
nit: my preference would be for spelling out the boolean e.g. `isDisabled={f.options.length === 0}`

Reply:
I can do that :)